lib/ecto_job/migrations.ex

```diff
@@ -65,11 +65,29 @@ defmodule EctoJob.Migrations do
       """)
     end
 
+    @doc"""
+    Upgrade jobs table to version v0.2.0, adding an error field to the first schema. Alter your
+    MyApp.Repo.Migrations.CreateJobQueue to:
+
+      def up do
+        table_name = "jobs"
+        EctoJob.Migrations.Install.up()
+        EctoJob.Migrations.CreateJobTable.up(table_name)
+        EctoJob.Migrations.CreateJobTable.upgrade(table_name, "v0.2.0")
+      end
+    """
+    def upgrade(name, "v0.2.0") do
```

Comment:
👍 I like the version number parameter here.
Can we add something similar to `up`, so that new users don't need to call `up` and `upgrade`?